CONTRIBUTING.md

```diff
@@ -0,0 +1,155 @@
+# DRAFT Contributing Guidelines
+
+Thank you for considering a contribution to the club-40 group project!
+
+## Got a Question?
+
+Please do not open issues for general support. We want to keep this issue tracker
+for bug reports and feature requests. Instead, we suggest you use the discussion feature in the repository.
+
+## Found a Bug?
+
+If you find a bug in the code, you can help us by submitting an issue.
+Or, better, submit a pull request with a fix readily available!
+
+## Missing a feature?
+
+You can request a new feature by submitting an issue to the issue tracker.
+If you would like to implement a new feature, please consider the impact it would have on the codebase:
+
+* For a major impact affecting multiple files, adding wholly new topics and sections to the documentation, etc.:
+    please open an issue first, outlining your proposal so that it can be discussed.
+    This allows us to better coordinate our resources around your proposal and help you in crafting your contribution.
+
+* Small features can be put together and directly submitted as a Pull Request.
+
+## Submission Guidelines
+
+### Submitting an Issue
+
+Before you submit a new issue, please search the issue tracker, maybe an issue relating to your problem already exists,
+and the discussion might inform you of fixes readily available.
+
+We want to fix all the issues as soon as possible, but before fixing a bug we need to reproduce and confirm it. Please provide as much information as possible in your initial post and be on the lookout for a response asking for additional information.
+
+### Submitting a Pull Request (PR)
+
+Before you submit your Pull Request, follow these steps:
+
+1. Search the repository for an open or closed PR that relates to your submission.
+   You wouldn't want to duplicate existing efforts.
+2. Be sure that an issue describes the problem you're fixing, or documents the design for the feature you're adding.
+   Discussing the design upfront helps to ensure that we're ready to accept your work.
+3. [Fork](https://github.com/Club-40/group-project-1 ) and then clone your fork.
+
+4. In your cloned repository, make your changes in a new branch:
+
+    ```shell
+    git checkout -b my-branch master
+    ```
+
+5. Make your changes, including tests when appropriate. Please refer to [`test/README.md`](TODO ) for a guide.
+
+6. Follow the current coding style as close as possible. Simply take a look at other files and stick to their styling.
+
+7. Commit your changes using a descriptive commit message following our [commit message conventions](TODO).
+
+8. Push your branch to GitHub:
+
+   ```shell
+   git push origin my-branch
+   ```
+
+9. On GitHub, send a pull request to `TODO`.
+
+#### Addressing review feedback
+
+If we ask for changes via code reviews then:
+
+1. Make the required updates to the code.
+
+1. Push the new changes to your repository (this will update your Pull Request as well)
```

Comment:
```suggestion
2. Push the new changes to your repository (this will update your Pull Request as well)
```

Reply:
should be making a new branch and cloning the main branch instead of forking it.
Forking is mostly done for open source contributions where you are not a member of the organization ( teams )
unless you guys have a different idea that I am not aware of or this is for the general public.